src/data/pnat/factors.lean

```diff
@@ -102,7 +102,7 @@ def prod (v : prime_multiset) : ℕ+ := (v : multiset pnat).prod
 theorem coe_prod (v : prime_multiset) : (v.prod : ℕ) = (v : multiset ℕ).prod :=
 begin
   let h : (v.prod : ℕ) = ((v.map coe).map coe).prod :=
-  (v.to_pnat_multiset.prod_hom coe).symm,
+  (v.to_pnat_multiset.prod_hom (monoid_hom.of coe)).symm,
```

Comment:
We have `monoid_hom.map_multiset_prod`.

Reply:
Okay. (This file is baffling to me, by the way. It spends the entire time making useless coercions and then fighting them...?)